core/actions/getReleaseNotes.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const async = require('async');
+
+module.exports = function getReleaseNotesBuilder(issueReleaseInfoList, releaseNotesFormatter) {
+  return (ids, filterLabels, callback) => {
+    async.waterfall([
+      (next) => {
+        issueReleaseInfoList.get(ids, next);
+      },
+
+      (releaseInfoList, next) => {
+        if (!filterLabels || filterLabels.length === 0) return next(null, releaseInfoList);
+
+        const filteredReleaseInfoList = releaseInfoList.filter(info => {
+          if (!info.issue || !info.issue.labels) return false;
+
+          var labels = info.issue.labels.map(labelInfo => labelInfo.name);
```

Comment:
It would be better if we avoid `var` since we are using ES2015. I think `const` work here 😉